xla/service/gpu/fusions/BUILD

```diff
@@ -663,11 +663,8 @@ cc_library(
 xla_test(
     name = "cudnn_test",
     srcs = if_cuda_is_configured(["cudnn_test.cc"]),
-    backend_tags = {"gpu": [
-        "requires-gpu-sm80",
-    ]},
     backends = [
-        "gpu",
+        "gpu_h100",
```

Comment:
The test is only intended for Hopper and does exactly nothing on Ampere: https://github.com/openxla/xla/blob/df736d78858f53bb957fd9c8db9d36af353d22e9/xla/service/gpu/fusions/cudnn_test.cc#L73. It was kept in the present state because tags like gpu_h100 did not work in OSS till yesterday.

Reply:
Thanks for clarifying, could you please change the existing sm80 to sm90 above instead of changing the backend here. That would automatically run the test in future generations.

Reply:
Done.

Reply:
Thanks.